.github/workflows/preview-docs.yml

```diff
@@ -1,146 +1,160 @@
+              uses: actions/setup-node@v4
+              with:
+                  node-version: 20
+                  cache: yarn
+
+            - name: Enable Corepack
+              run: corepack enable
+
+            - name: Generate PR hash
+              id: pr_hash
+              run: |
+                  # Use just PR number so each commit overwrites the previous deployment
+                  pr_hash="pr-${{ github.event.pull_request.number }}"
+                  echo "hash=${pr_hash}" >> "$GITHUB_OUTPUT"
+                  echo "Generated PR hash: ${pr_hash}"
+
+            ## --- YARN CACHE --- ##
+            - name: Check for cached dependencies
+              continue-on-error: true
+              id: cache-dependencies
+              uses: actions/cache@v4
+              with:
+                  path: |
+                      .cache/yarn
+                      node_modules
+                  key: ubuntu-latest-node20-${{ hashFiles('yarn.lock', 'package.json') }}
+
+            ## --- INSTALL & BUILD --- ##
+            - name: Install dependencies
+              shell: bash
+              run: yarn install --immutable
+
+            - name: Build all generations
+              run: yarn build
+
+            - name: Generate Custom Elements Manifest
+              run: yarn workspace @spectrum-web-components/1st-gen docs:analyze
+
+            - name: Move CEM to Storybook directory
+              run: cp 1st-gen/projects/documentation/custom-elements.json 1st-gen/storybook/
+
+            - name: Build documentation with path prefix
+              env:
+                  SWC_DIR: ${{ steps.pr_hash.outputs.hash }}/docs/first-gen-docs
+              run: |
+                  yarn workspace @spectrum-web-components/1st-gen docs:build
+
+            - name: Build first-gen Storybook
+              run: yarn workspace @spectrum-web-components/1st-gen storybook:build
+
+            - name: Build second-gen Storybook
+              run: yarn workspace @spectrum-web-components/2nd-gen storybook:build
+
+            ## --- DEPLOY TO AZURE BLOB STORAGE --- ##
+            - name: Setup AzCopy
+              uses: ./.github/actions/setup-azcopy
+
+            - name: Deploy to Azure Blob Storage
+              id: deploy
+              env:
+                  PR_HASH: ${{ steps.pr_hash.outputs.hash }}
+              run: |
+                  # Upload documentation
+                  echo "Uploading first-gen documentation to ${PR_HASH}/docs/first-gen-docs/"
+                  azcopy copy "1st-gen/projects/documentation/dist/*" \
+                    "https://swcpreviews.blob.core.windows.net/\$web/${PR_HASH}/docs/first-gen-docs/" \
+                    --recursive \
+                    --from-to LocalBlob
+
+                  # Upload first-gen Storybook
+                  echo "Uploading first-gen Storybook to ${PR_HASH}/docs/first-gen-storybook/"
+                  azcopy copy "1st-gen/projects/documentation/dist/storybook/*" \
```

Comment:
Can you tell me again- isn't there a line that I can comment out so that I can see all of the hidden files? Where is that once more?